ec2/sc-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -373,6 +372,9 @@ Resources:
           Value: "yes"
         - Key: "PatchGroup"
           Value: "prod-default"
+    CreationPolicy:
+      ResourceSignal:
+        Timeout: PT10M
```

Comment:
do you think this is enough of a timeout for notebook?

Reply:
The duration was fine during testing, but I've increased it to 20 as I don't think it will harm anything.